Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -277,14 +276,11 @@ private static InputOverlayDrawableJoystick initializeOverlayJoystick(Context co
 		Rect outerRect = new Rect(drawableX, drawableY, drawableX + outerSize, drawableY + outerSize);
 		Rect innerRect = new Rect(0, 0, outerSize / 4, outerSize / 4);
 
-		final InputOverlayDrawableJoystick overlayDrawable
-				= new InputOverlayDrawableJoystick(res,
-					bitmapOuter, bitmapInner,
-					outerRect, innerRect,
-					joystick);
 
-
-		return overlayDrawable;
+		return new InputOverlayDrawableJoystick(res,
+            bitmapOuter, bitmapInner,
```

Comment:
Indentation mismatch here. We use tabs for indenting and spaces for vertical alignment